### ISSUE TITLE
src/accessibility.ts:41:5 - error TS1117: An object literal cannot have multiple properties with the same name.

### ISSUE THREAD

**Issue by joshholly** [AmrDab/clawdcursor#10]
I tried to just uninstall and reinstall, but I get this when I try to build

```
src/accessibility.ts:41:5 - error TS1117: An object literal cannot have multiple properties with the same name.

41     'get-screen-context': 'get-screen-context.jxa',
```

**Comment by AmrDab** [comment]
Fixed. There was a duplicate get-screen-context entry in the darwin script map that caused the TS build error.
 Already pushed to main. git pull origin main and rebuild should work now.
